README.md

```diff
@@ -1,25 +1,16 @@
-# Robot-Framework V2
-
-This repo is meant to be used as a template for robots made for [OpenOrchestrator](https://github.com/itk-dev-rpa/OpenOrchestrator).
+# Masseoprettelse i KMD Nova
+An Open Orchestrator RPA solution for reading emails sent from OS2 Forms and outputting notes in KMD Nova.
 
 ## Quick start
 
-1. To use this template simply use this repo as a template (see [Creating a repository from a template](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-repository-from-a-template)).
-__Don't__ include all branches.
-
-2. Go to `robot_framework/__main__.py` and choose between the linear framework or queue based framework.
-
-3. Implement all functions in the files:
-    * `robot_framework/initialize.py`
-    * `robot_framework/reset.py`
-    * `robot_framework/process.py`
-
-4. Change `config.py` to your needs.
+Install the robot in Open Orchestrator, with a list of accepted AZ idents for people who should be allowed to input data from OS2 Forms:
 
-5. Fill out the dependencies in the `pyproject.toml` file with all packages needed by the robot.
-
-6. Feel free to add more files as needed. Remember that any additional python files must
-be located in the folder `robot_framework` or a subfolder of it.
+```json
+{
+    "accepted_azs" : ["az00000", "az000001"]
+}
+```
+Setup email and link to Open Orchestrator queue in config.py and setup Nova access in Open Orchestrator credentials.
```

Comment:
Consider using https://github.com/igorshubovych/markdownlint-cli to clean up Markdown files and keep them consistent.

```suggestion

Setup email and link to Open Orchestrator queue in config.py and setup Nova access in Open Orchestrator credentials.
```